### ISSUE TITLE
TypeError: Do not know how to serialize a BigInt

### ISSUE THREAD

**Issue by Dexconv** [wighawag/hardhat-deploy#457]
**Describe the bug**
when trying to run a hardhat node with my deployment scripts, i get the following error:
```powershell
/.../hardhat-erc20/node_modules/hardhat-deploy/src/DeploymentsManager.ts:861
      JSON.stringify({
           ^
TypeError: Do not know how to serialize a BigInt
    at JSON.stringify (<anonymous>)
    at DeploymentsManager.saveDeployment (/.../hardhat-erc20/node_modules/hardhat-deploy/src/DeploymentsManager.ts:861:12)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at runNextTicks (node:internal/process/task_queues:64:3)
    at listOnTimeout (node:internal/timers:533:9)
    at processTimers (node:internal/timers:507:7)
``` 

when using hardhat deploy, the contract is deployed correctly and the address is returned, also when i use hardhat node, after returning the contract address and accounts this error is shown and the node closes
i downgraded ethers to  5.7.2 and the issue was fixed but with the current version of 6.4.0 the issue persists

my deploy script:
```javascript
const { network, ethers } = require("hardhat")
const { developmentChains } = require("../helper-hardhat.config")

const initial_supply = ethers.parseEther("50")

module.exports = async ({ getNamedAccounts, deployments, utils }) => {
    const { deployer } = await getNamedAccounts()
    const { deploy, log } = deployments

    if (developmentChains.includes(network.name)) {
        log("development chain found")
        log("deploying with the initial supply of: ", initial_supply)

        //deploy the contract
        await deploy("OurToken", {
            args: [initial_supply],
            from: deployer,
            log: true,
        })
    }
}
``` 

**Expected behavior**
to run the hardhat node with the latest versions

**versions**
 - hardhat-deploy "0.11.30
 - hardhat "2.15.0"
 - ethers "6.4.0"
 - nodejs v18.12.1

**Comment by wighawag** [comment]
I published a potential fix for it on v0.11.31

Could you try it and report back, thanks!

**Comment by Dexconv** [comment]
perfect!
i tried it out with v0.11.31 and it workes, thanks for the fix
___________
also figured out by changing  the 3rd line to
```javascript
const initial_supply = ethers.parseEther("50").toString()
```
it also works
